plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -905,4 +892,10 @@ public void setInstalledPluginVersions(Map<String, VersionNumber> installedPlugi
     public void setBundledPluginVersions(Map<String, VersionNumber> bundledPlugins) {
         bundledPluginVersions = bundledPlugins;
     }
+
+    public void log(String message) {
```

Comment:
I think this should be `verbose` or `logVerbose` or `log(message, verbose)`
log isn't what it is doing